sktime/forecasting/tests/test_pytorchforecasting.py

```diff
@@ -0,0 +1,96 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Tests for interfacing estimators from pytorch-forecasting."""
+
+import pytest
+
+from sktime.datatypes._utilities import get_cutoff
+from sktime.forecasting.base._fh import ForecastingHorizon
+from sktime.forecasting.pytorchforecasting import (
+    PytorchForecastingDeepAR,
+    PytorchForecastingNBeats,
+    PytorchForecastingNHiTS,
+    PytorchForecastingTFT,
+)
+from sktime.tests.test_switch import run_test_for_class
+from sktime.utils._testing.forecasting import (
+    _assert_correct_columns,
+    _assert_correct_pred_time_index,
+)
+from sktime.utils._testing.hierarchical import _make_hierarchical
+
+__author__ = ["XinyuWu"]
+
+
+@pytest.mark.parametrize(
+    "model_class",
+    [
+        PytorchForecastingDeepAR,
+        PytorchForecastingNBeats,
+        PytorchForecastingNHiTS,
+        PytorchForecastingTFT,
+    ],
+)
+@pytest.mark.skipif(
+    not run_test_for_class(
+        [
+            PytorchForecastingDeepAR,
+            PytorchForecastingNBeats,
+            PytorchForecastingNHiTS,
+            PytorchForecastingTFT,
+        ]
+    ),
+    reason="run test only if softdeps are present and incrementally (if requested)",
+)
+def test_load_model_from_disk(model_class) -> None:
+    """Test load fitted model from disk with refit."""
+    # define model
+    model = model_class(**(model_class.get_test_params()[0]))
+
+    # generate data
+    data_length = 100
+    data = _make_hierarchical(
+        (5, 100),
+        n_columns=2,
+        max_timepoints=data_length,
+        min_timepoints=data_length,
+    )
+    l1 = data.index.get_level_values(1).map(lambda x: int(x[3:]))
+    X_train = data.loc[l1 < 90, "c0"].to_frame()
```

Comment:
Also here please check if this is the same as InstanceSplitter is doing.

Reply:
replaced with `sklearn.model_selection.train_test_split`